addons/mail/static/tests/message/message.test.js

```diff
@@ -225,8 +225,7 @@ test.skip("Can edit message comment in chatter (mobile)", async () => {
     });
     await start();
     await openFormView("res.partner", partnerId);
-    await click(".o-mail-Message [title='Expand']");
-    await click("button:contains('Edit')");
+    await click(".o-mail-Message [title='Edit']");
     await contains("button", { text: "Discard editing" });
     await insertText(".o-mail-Message .o-mail-Composer-input", "edited message", { replace: true });
     await click("button[title='Save editing']");
```

Comment:
Test has "(mobile)" in name but it's not actually run in mobile. This is shown by this interaction on `Save editing` that is specific to desktop.

Should move to `mobile.test.js` (to have `"mobile" tag`), and also should `mockUserAgent("android");` & `mockTouch(true);` to properly have `isMobileOS()` environment for the good working of test.

Reply:
Also don't forget to unskip the test ;)